pkg/backup/backupapi/types.go

```diff
@@ -21,16 +21,19 @@ type ServiceStatus struct {
 	// the backup service
 	RecentBackup *BackupStatus `json:"recentBackup,omitempty"`
 
-	// Backups is the totoal number of existing backups
+	// Backups is the totoal number of existing backups.
 	Backups int `json:"backups"`
+
+	// BackupSize is the total size of existing backups in MB.
+	BackupSize float64 `json:"backupSize"`
```

Comment:
we have size in backup status struct. i want to keep them consistent.

i also feel adding inMB is better. However, we cannot really change the one in backup status for compatibility reason. I do not think this is a big deal either.